pkgs/tools/misc/minipro/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, pkg-config
+, libusb1
+, which
+}:
+
+stdenv.mkDerivation rec {
+  pname = "minipro";
+  version = "0.5";
+
+  src = fetchFromGitLab {
+    owner = "DavidGriffith";
+    repo = "minipro";
+    rev = version;
+    sha256 = "sha256-Hyj2LyY7W8opjigH+QLHHbDyelC0LMgGgdN+u3nNoJc=";
+  };
+
+  nativeBuildInputs = [ pkg-config which ];
+  buildInputs = [ libusb1 ];
+  makeFlags = [ "DESTDIR=$(out)" "PREFIX=" "CC=cc" ];
+
+  meta = with lib; {
+    homepage = "https://gitlab.com/DavidGriffith/minipro";
+    description = "An open source program for controlling the MiniPRO TL866xx series of chip programmers";
+    license = licenses.gpl3;
```

The license looks like [`GPLv3+`](https://gitlab.com/DavidGriffith/minipro/-/blob/master/ihex.c#L6-9).

```suggestion
    license = licenses.gpl3Plus;
```